### ISSUE TITLE
Degree of establishment should not be `eradicated` nor `transported`

### ISSUE THREAD

**Issue by LienReyserhove** [trias-project/alien-mammals-checklist#3]
For `Castor canadensis Kuhl, 1820`, the degree of establishment is `eradicated`. This is not a valid value, we need to stick to the [vocabulary for degreeOfEstablishment](https://rs.gbif.org/vocabulary/dwc/degree_of_establishment_2022-02-02.xml
). Moreover, if it's eradicated, I guess it's not present anymore, but rather absent?

Similar, the degree of establishment for `Eptesicus fuscus (Palisot de Beauvois, 1796)` is `transported`. This should be corrected as well.

**Comment by qgroom** [comment]
There status refers to the period between the first and last record, not necessarily the current data, so shouldn't it be present?

**Comment by LienReyserhove** [comment]
Agree to keep it `present` 👍

**Comment by sannegovaert** [comment]
@timadriaens, can you look into this?
For _Castor canadensis_, I propose to fill in `date of last observation` and change `degree of establishment`.
For _Eptesicus fuscus_, change `degree of establishment`.

**Comment by timadriaens** [comment]
_Castor canadensis_: The first North American beaver in Belgium was recorded in 2001 (Lejeune 2021). By 2009, multiple sites had been confirmed over a larger territory, all of which were close to the border with Germany or Luxembourg. It is therefore likely that they too spread from the bordering state of Rhineland-Palatinate (Germany). Following control actions, which removed tens of specimens (see Qu. 2.5) and coincided with a natural expansion of C. fiber, the presence of C. canadensis has drastically declined up to the point that it is now probably extinct. >> this sounds more like `degree of establishment` is **casual**. Last record in OFFH is from 2023. I added this in the checklist. 

Is there no other Darwin core field that can be populated with "eradicated"? It is the same for Cervus nippon.

_Eptesicus fuscus_ is problematic, actually it was found as a transport contaminant in a shipment of wood from Charleston. "Transported" would be better but this is not a term in the vocab. I'd go for **captive** as it has not "left the pathway". Changed this in the checklist.
